calicoctl/calico_ctl/node.py

```diff
@@ -574,6 +575,9 @@ def node_stop(force):
     Stop the Calico node.  This stops the containers (calico/node and
     calico/node-libnetwork) that are started by calicoctl node.
     """
+    # The command has to be run as root to stop the calico-node service
```

Comment:
Any idea why the systemctl command is in here.  Seems kind of odd to me that we are putting service stuff in this command.